lib/tasks/users.rake

```diff
@@ -0,0 +1,27 @@
+require 'csv'
+
+namespace :ofn do
+  desc 'remove the limit of enterprises a user can create'
+  task :remove_enterprise_limit, [:user_id] => :environment do |_task, args|
+    RemoveEnterpriseLimit.new(args.user_id).call
```

Comment:
Just out of curiosity, why do you store the user id in the service object instead of just doing `RemoveEnterpriseLimit.call(args.user_id)`?

Reply:
Just inertia. I'm used to do so as most of the times classes tend to be more complex than this one.